### ISSUE TITLE
How to set popup align?

### ISSUE THREAD

**Issue by jaksab** [ViHtarb/Tooltip#16]
Can I set popup align relatively target view (like in library https://android-arsenal.com/details/1/3826)?
Now i can use only Gravity.BOTTOM, but i want to set right align, that popup is shown on the lower right.
Can you help me?

**Comment by ViHtarb** [comment]
Yes you can. Use it:
`Builder` method
```java
.setGravity(Gravity.TOP)
```
Example:
```java
Tooltip.Builder builder = new Tooltip.Builder(v)
        .setGravity(Gravity.TOP)
        .setText("Hello World!");
```
or style attribute
```xml
<attr name="android:gravity"/>
```
Example:
```java
Tooltip.Builder builder = new Tooltip.Builder(v, R.style.Tooltip);
```
```xml
<style name="Tooltip">
    <item name="android:text">Hello World!</item>
    <item name="android:gravity">top</item>
</style>
```
PS: All library classes methods and styleable attributes described in [Readme](https://github.com/ViHtarb/Tooltip/blob/master/README.md)

**Comment by jaksab** [comment]
Yes, do it please.

**Comment by jaksab** [comment]
Will be whether the update?

**Comment by ViHtarb** [comment]
@jaksab yes, there is, but at the moment I'm very busy, as I can, as soon as'll develop these feature

**Comment by JoaquimLey** [comment]
Any news on this @ViHtarb ?

**Comment by ViHtarb** [comment]
@JoaquimLey no :( sorry but I'm very busy now. I continue to develop library in free time but very slowly

**Comment by wonsuc** [comment]
I think this is most important feature.

**Comment by kassim** [comment]
still not present :(

**Comment by swatiomar** [comment]
Also I want to right align.
Any update on This @ViHtarb
